spec.md

```diff
@@ -210,11 +210,39 @@ help intermediate gateways determine how to route the events.
 * Constraints:
   * REQUIRED
 * Examples
-    * https://github.com/cloudevents/spec/pull/123
-    * /cloudevents/spec/pull/123
-    * urn:event:from:myapi/resourse/123
+    * https://github.com/cloudevents/spec/pull
+    * /cloudevents/spec/pull
+    * urn:event:from:myapi/resource
     * mailto:cncf-wg-serverless@lists.cncf.io
 
+### subject
+* Type: `String`
+* Description: This describes the subject of the event in the context of the
+  event producer (identified by `source`). In publish-subscribe scenarios,
+  a subscriber will typically subscribe to events emitted by a `source`,
+  but the `source` identifier alone might not be sufficient as a qualifier for
+  any specific event if the `source` context has internal sub-structure.
+
+  Identifying the subject of the event in context metadata (opposed to only in
+  the `data` payload) is particularly helpful in generic subscription filtering
+  scenarios where middleware is unable to interpret the `data` content. In the
+  above example, the subscriber might only be interested in blobs with names
+  ending with '.jpg' or '.jpeg' and the subject attribute allows for 
+  constructing a simple and efficient string-suffix filter for that subset of
+  events.
+* Constraints:
+  * OPTIONAL
+  * MUST be a non-empty string
+* Example:
```

Comment:
I like this example because it shows how all of the key properties relate to each other, and better explains what people should expect in each. So, can we move this example out from under the definition of this one property so it can be view in the bigger context?  And then show "id" and "type" too - or even a total CE.